XLS-0056d-batch/README.md

```diff
@@ -0,0 +1,626 @@
+      "0C4316F7E7D909E11BB7DBE0EB897788835519E9950AE8E32F5182468361FE7E"
+    ],
+    BatchSigners: [
+      {
+        BatchSigner: {
+          Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+          SigningPubKey: "03072BBE5F93D4906FC31A690A2C269F2B9A56D60DA9C2C6C0D88FB51B644C6F94",
+          Signature: "304502210083DF12FA60E2E743643889195DC42C10F62F0DE0A362330C32BBEC4D3881EECD022010579A01E052C4E587E70E5601D2F3846984DB9B16B9EBA05BAD7B51F912B899"
+        }
+      },
+      {
+        BatchSigner: {
+          Account: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+          SigningPubKey: "03C6AE25CD44323D52D28D7DE95598E6ABF953EECC9ABF767F13C21D421C034FAB",
+          Signature: "304502210083DF12FA60E2E743643889195DC42C10F62F0DE0A362330C32BBEC4D3881EECD022010579A01E052C4E587E70E5601D2F3846984DB9B16B9EBA05BAD7B51F912B899"
+        }
+      },
+    ],
+    Sequence: 4,
+    Fee: "60",
+    SigningPubKey: "03072BBE5F93D4906FC31A690A2C269F2B9A56D60DA9C2C6C0D88FB51B644C6F94",
+    TxnSignature: "30440220702ABC11419AD4940969CC32EB4D1BFDBFCA651F064F30D6E1646D74FBFC493902204E5B451B447B0F69904127F04FE71634BD825A8970B9467871DA89EEC4B021F8"
+  },
+  {
+    TransactionType: "Payment",
+    Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+    Destination: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+    Amount: "6000000",
+    BatchTxn: {
+      OuterAccount: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+      Sequence: 4,
+      BatchIndex: 0
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  },
+  {
+    TransactionType: "Payment",
+    Account: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+    Destination: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+    Amount: {
+      currency: "GKO",
+      issuer: "ruazs5h1qEsqpke88pcqnaseXdm6od2xc",
+      value: "2"
+    },
+    BatchTxn: {
+      OuterAccount: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+      Sequence: 20,
+      BatchIndex: 0
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  }
+]
+```
+</details>
+
+# Appendix
+
+## Appendix A: FAQ
+
+### A.1: What if I want a more complex tree of transactions that are AND/XORed together? Can I nest `Batch` transactions?
+
+The original version of this spec supported nesting `Batch` transactions. However, upon further analysis, that was deemed a bit too complicated for an initial version of `Batch`, as most of the benefit from this new feature does not require nested transactions. Based on user and community need, this could be added as part of a V2.
+
+### A.2: What if all of the transactions fail? Will a fee still be claimed?
+
+Yes, just as they would if they were individually submitted.
+
+### A.3: Would this feature enable greater frontrunning abilities?
+
+That is definitely a concern. Ways to mitigate this are still being investigated. Some potential answers:
+* Charge for more extensive path usage
+* Have higher fees for `Batch` transactions
+* Submit the `Batch` transactions at the end of the ledger
```

Comment:
Naively, this sounds like it would work, but in practice it might make it even easier to front-run since you could submit a mix of Batch and non-Batch transactions to more effectively "sandwich" a targeted trade.